src/Coff.zig

```diff
@@ -0,0 +1,59 @@
+const Coff = @This();
+
+const std = @import("std");
+const assert = std.debug.assert;
+const coff = std.coff;
+const fs = std.fs;
+const log = std.log.scoped(.coff);
+const mem = std.mem;
+
+const Allocator = mem.Allocator;
+const Zld = @import("Zld.zig");
+
+pub const base_tag = Zld.Tag.coff;
+
+base: Zld,
+
+pub fn openPath(allocator: *Allocator, options: Zld.Options) !*Coff {
+    const file = try options.emit.directory.createFile(options.emit.sub_path, .{
+        .truncate = true,
+        .read = true,
+        .mode = 0,
```

Comment:
I think this should include a check like in other backends when using `zld` to cross-link to a foreign platform from Linux say.

```suggestion
        .mode = if (std.Target.current.os.tag == .windows) 0 else 0o777,
```

Reply:
Sure, that's fixed now. I thought it would only be used on Windows platforms.

Reply:
So the idea is that in principle since `zld` is written in Zig, it can easily be cross-compiled for any platform which we should make use by giving the possibility to easily cross-link to any platform too. It will obviously be lacking in this department all of the automation tooling that goes with Zig proper, but this where once would use the `zig ld` abstraction embedded directly in Zig toolchain.

Reply:
Also, I'm not sure how this works with cross-linking to Windows from Linux while testing in Wine. Something I need to give a look but didn't yet. It might not be needed indeed, but for consistency I'd like for all linker backends to set the mode correctly :-)